generators/app/templates/src/utils/array.js

```diff
@@ -4,10 +4,14 @@ import Immutable from 'seamless-immutable';
  * Receives an array of strings, and returns an obj with that strings as properties with that string as value.
  * E.G:
  * stringArrayToObject(['A', 'B', 'C']) // { A: 'A', B: 'B', C: 'C' }
+ * @param {array} actionsArray array of values
+ * @param {string} namespace prefix for the resulting values
+ * @returns {object} (['A', 'B', 'C', 'name']) -> { A: '@name/A', B: '@name/B', C: '@name/C' }
  */
 export function stringArrayToObject(actionsArray, namespace = '') {
```

Comment:
`createTypes` from `r-r` is the same function 😜

Reply:
r-r is an optional dependency... we could make it so it imports this or r-r dependingof which options the user chooses, but it's beyond this PR's scope. I'm only settings the linter and fixing it's issues here.

Reply:
I know, I know 😅